src/libs/elektra/kdb.c

```diff
@@ -1222,20 +1242,24 @@ int kdbSet (KDB * handle, KeySet * ks, Key * parentKey)
 		copyError (parentKey, oldError);
```

Comment:
can not comment in line 1228 so i put it here:
There is a return in 1228, should there not be before the return call some hook calls (`postgetstorage`)?